README.md

```diff
@@ -1,463 +1,475 @@
+Parse a cookie date string into a `Date`. Parses according to RFC 6265 Section 5.1.1, not `Date.parse()`.
 
-### `formatDate(date)`
+#### `formatDate(date)`
 
-Format a Date into a RFC1123 string (the RFC6265-recommended format).
+Format a `Date` into an RFC 1123 string (the RFC 6265 recommended format).
 
-### `canonicalDomain(str)`
+#### `canonicalDomain(str)`
 
-Transforms a domain-name into a canonical domain-name.  The canonical domain-name is a trimmed, lowercased, stripped-of-leading-dot and optionally punycode-encoded domain-name (Section 5.1.2 of RFC6265).  For the most part, this function is idempotent (can be run again on its output without ill effects).
+Transforms a domain name into a canonical domain name. The canonical domain name is a domain name that has been trimmed, lowercased, stripped of leading dot, and optionally punycode-encoded (Section 5.1.2 of RFC 6265). For the most part, this function is idempotent (calling the function with the output from a previous call returns the same output).
 
-### `domainMatch(str,domStr[,canonicalize=true])`
+#### `domainMatch(str, domStr[, canonicalize=true])`
 
-Answers "does this real domain match the domain in a cookie?".  The `str` is the "current" domain-name and the `domStr` is the "cookie" domain-name.  Matches according to RFC6265 Section 5.1.3, but it helps to think of it as a "suffix match".
+Answers "does this real domain match the domain in a cookie?". The `str` is the "current" domain name and the `domStr` is the "cookie" domain name. Matches according to RFC 6265 Section 5.1.3, but it helps to think of it as a "suffix match".
 
-The `canonicalize` parameter will run the other two parameters through `canonicalDomain` or not.
+The `canonicalize` parameter toggles whether the domain parameters get normalized with `canonicalDomain` or not.
 
-### `defaultPath(path)`
+#### `defaultPath(path)`
 
-Given a current request/response path, gives the Path apropriate for storing in a cookie.  This is basically the "directory" of a "file" in the path, but is specified by Section 5.1.4 of the RFC.
+Given a current request/response path, gives the path appropriate for storing in a cookie. This is basically the "directory" of a "file" in the path, but is specified by Section 5.1.4 of the RFC.
 
-The `path` parameter MUST be _only_ the pathname part of a URI (i.e. excludes the hostname, query, fragment, etc.).  This is the `.pathname` property of node's `uri.parse()` output.
+The `path` parameter MUST be _only_ the pathname part of a URI (excluding the hostname, query, fragment, and so on.). This is the `.pathname` property of node's `uri.parse()` output.
 
-### `pathMatch(reqPath,cookiePath)`
+#### `pathMatch(reqPath, cookiePath)`
 
-Answers "does the request-path path-match a given cookie-path?" as per RFC6265 Section 5.1.4.  Returns a boolean.
+Answers "does the request-path path-match a given cookie-path?" as per RFC 6265 Section 5.1.4. Returns a boolean.
```

Comment:
Include link to [Section 5.1.4](https://datatracker.ietf.org/doc/html/rfc6265#section-5.1.4) of the RFC